src/browser/api/window.js

```diff
@@ -502,6 +502,35 @@ Window.create = function(id, opts) {
             }
         });
 
+        const emitToAppAndWin = (...types) => {
+            let isMainWindow = (uuid === name);
```

Comment:
yeah i agree as a first step there should be one centralized `isMainWindow` core method even if, today, it still uses `uuid === name`... not sure if this is _that_ ticket though...?